runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -6099,6 +6099,49 @@ TR_InlinerFailureReason
       return Recognized_Callee;
       }
 
+   // This logic that decides whether to inline or accelerate hasNegatives and countPositives only applies to x86
```

Comment:
Thanks for adding that comment, but I think you further need to explain why it only applies to x86.  I hadn't looked at this code for a few weeks and it was already lost on me why that's the case.